src/tv2-common/helpers/graphics/pilot/PilotGraphicGenerator.ts

```diff
@@ -184,6 +191,33 @@ export abstract class PilotGraphicGenerator extends Graphic {
 			: this.config.studio.VizPilotGraphics.PrerollDuration
 	}
 
+	protected getTv2PieceType(): Tv2PieceType {
+		switch (this.engine) {
+			case 'OVL':
+				return Tv2PieceType.OVERLAY_GRAPHICS
+			case 'WALL':
+			case 'FULL':
+			case 'TLF':
+				return Tv2PieceType.GRAPHICS
+			default:
+				return Tv2PieceType.GRAPHICS
+		}
+	}
+
+	protected getTv2OutputLayer(): Tv2OutputLayer | undefined {
+		switch (this.engine) {
+			case 'WALL':
+				return undefined
```

Comment:
The WALL case can be removed or moved down to the default case without changing the behaviour.

Reply:
I have removed the case like you suggested.